utils/fork-tree/src/lib.rs

```diff
@@ -939,6 +944,10 @@ mod test {
 		//   — J - K
 		//
 		// (where N is not a part of fork tree)
+		//
+		// NOTE: the tree will get automatically rebalance on import and won't be laid out like the
+		// diagram above. the children will be ordered by subtree depth and the longest branches
+		// will be on the leftmost side of the tree.
```

Comment:
I was thinking a post rebalance diagram would be good, but the explanation is enough (I tried to update the schema but got lost with my bad ascii writing whent on the other hand the iter test was easy to read from the old schema and this comment).

Reply:
Yeah I was thinking of doing the same thing but then opted to just add the comment instead :P